app/services/symphony_client.rb

```diff
@@ -65,7 +69,10 @@ def patron_info(patron_key, item_details: {})
     })
 
     JSON.parse(response.body)
+  rescue StandardError
```

Comment:
Can we rescue something more specific than `StandardError` here? What is the thing that actually fails in this scenario?

Reply:
I actually removed this check because after @shelleydoljack and I looked into the problem deeper, we are not sure that this is where to do the check, so we decided to leave this out of this PR and try to address the issue on a different PR to for #501 and #495

Reply:
Yes, this can be `JSON::ParserError`